### ISSUE TITLE
Why My CTC Loss is always Nan or Inf value?

### ISSUE THREAD

**Issue by hemanthyernagula** [meijieru/crnn.pytorch#235]
While training the model at every epoch the loss is always a nan or inf value, I tried to debug but could not sort out. If anyone face this kind of issue please help me out. 

![image](https://user-images.githubusercontent.com/30042812/96090797-63872400-0ee6-11eb-8297-6cf11bcada0e.png)

**Comment by zuoyuwei** [comment]
encounter the same question, ask for help

**Comment by RobertXiaobinW** [comment]
Have you already solved the problem?

**Comment by gittjob** [comment]
I'm in the same situation. Have you solved it?

**Comment by hemanthyernagula** [comment]
No I did not

**Comment by GraceKafuu** [comment]
Same problem

**Comment by Yuxiang1995** [comment]
1. make sure all the labels are not empty
2. make sure the length of the predicted sequence > 2 * length(label) + 1
